src/main/java/com/wcc/platform/service/PlatformService.java

```diff
@@ -0,0 +1,18 @@
+package com.wcc.platform.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/** Platform service. */
+@Service
+public class PlatformService {
+  private final ObjectMapper objectMapper;
```

Comment:
```suggestion
  private final MemberRepository repository;
```

Reply:
So I do not use ObjectMapper? Does this mean that I have to create a new configuration MemberRepositoryConfig, in way it is done in the ObjectMapperConfig class?

Reply:
I am using objectMapper [here](https://github.com/Women-Coding-Community/wcc-backend/pull/39/files#diff-91c0d512d7d9d1c121a3f4cbcab46a95a7c8cb83dce0bd40376a37a1fb5604c7) -  shall I continue like this or this will be changed too?